video/out/hwdec/hwdec_cuda.c

```diff
@@ -76,17 +104,8 @@ static int cuda_init(struct ra_hwdec *hw)
     CudaFunctions *cu;
     int level = hw->probing ? MSGL_V : MSGL_ERR;
 
-    ret = cuda_load_functions(&p->cu, NULL);
-    if (ret != 0) {
-        MP_MSG(hw, level, "Failed to load CUDA symbols\n");
-        return -1;
-    }
     cu = p->cu;
```

Comment:
this has to be moved, `p->cu` is now NULL at this point

Reply:
No longer applicable with 3b77fccfc804e3e565864f8da289d2034dc19245